modules/nf-core/umicollapse/main.nf

```diff
@@ -33,7 +33,7 @@ process UMICOLLAPSE {
     # by conda that allows to set the heap size (Xmx), but not the stack size (Xss).
     # `which` allows us to get the directory that contains `umicollapse`, independent of wheather we
     # are in a container or conda environment.
-    UMICOLLAPSE_JAR=$(dirname $(which umicollapse))/../share/umicollapse-${VERSION}/umicollapse.jar
+    UMICOLLAPSE_JAR=\$(dirname \$(which umicollapse))/../share/umicollapse-${VERSION}/umicollapse.jar
```

Comment:
Don't you have to escape the `$` for `${Version}` as well then?

Also mind that I had already added the `-Xmx` parameter, so you need to remove my declaration then.

Reply:
VERSION is a groovy variable, so no ;) 
But good catch with the double Xmx!